### DISCUSSION TITLE
Removing [BOT] tag from TAB view not possible?

### DISCUSSION THREAD

**Discussion by akiraic** [sjavi4/PlayerDoll#85] [Q&A] [answered]
Greetings!

I'm testing the plugin with LuckPerms and Essentials installed, but none can take control of the [BOT] prefix for the doll name in the scoreboard (holding TAB). I didn't find anything in the documentation or the configuration files. Can you clarify if it is possible to remove it and if so, how?

Server version: 1.21, Paper
Plugins: Essentials X, LuckyPerms

Thank you very much, and I appreciate your efforts.

![image](https://github.com/user-attachments/assets/e5523464-adad-4c96-a60c-6533a32286c7)

**Comment by sjavi4** [comment, marked as answer]
https://github.com/sjavi4/PlayerDoll/wiki/Configs#perm-config

If you enabled perm.yml, you can change the prefix and suffix for specific permission holder.

**Reply by akiraic** [reply]
I was blind. I should have read it more carefully. Thank you. This worked!
